test/kafka-matrix/mzworkflows.py

```diff
@@ -0,0 +1,56 @@
+# Copyright Materialize, Inc. and contributors. All rights reserved.
+#
+# Use of this software is governed by the Business Source License
+# included in the LICENSE file at the root of this repository.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0.
+
+from materialize.mzcompose import (
+    Kafka,
+    Materialized,
+    SchemaRegistry,
+    Testdrive,
+    Workflow,
+    Zookeeper,
+)
+
+CONFLUENT_PLATFORM_VERSIONS = [
+    "4.0.0",
+    "5.0.0",
+    "6.0.2",
+    "6.1.2",
+    "6.2.0",
+    "latest",
+]
+
+services = [
```

Comment:
is this a special ident that mzworkflow expects to import?

Reply:
Yup! There's documentation on all this here! https://github.com/MaterializeInc/materialize/blob/main/doc/developer/mzworkflows.md